docs/UserGuide.md

```diff
@@ -313,32 +337,37 @@ Examples:
 Possible Usage:
 
 * You can use `show` to display the assignments of the student you would like to see and make modifications to.
+* You can look to the top of the assignment list to find out which assignments will be due soon or are overdue.
 
 ### Giving assignments: `give` `giveall`
 
 #### Giving an assignment to a student: `give`
 
-Gives an assignment with a deadline to the student specified by the index in the contact list.
+Gives an assignment to the student specified by the index in the contact list.
 
 Format: `give INDEX d/DESCRIPTION by/ d/M/yyyy [,HHmm]`
 
-<div markdown="span" class="alert alert-primary">
+* Gives the specified assignment to the student in the contact list at the specified `INDEX`.
 
-:bulb: **Tip:**
+<div markdown="span" class="alert alert-primary">
 
-* You can give assignments with due dates that has passed the current date of use.
+:bulb: **Tip:**<br>
 
 * The date `d/M/yyyy` can be replaced by [friendly commands](#friendly-commands).
 
 * If time `[,HHmm]` is not specified, time will be set to `11:59 pm` by default.
+
+* You can give assignments with due dates before today in case you wish to keep a record of old assignments.
 </div>
 
-**Note:**
-1. It is possible to add an assignment using `giveall` even if some students have the assignment already. Students who have the
-assignment will not receive a duplicate assignment.
-2. When using `giveall` in the situation stated in point 1, the specified assignment in the input needs to have the same
-due date as the existing assignment. **The command will not be valid if the due date is different.** This is to ensure
-consistency of the due dates of the same assignment.
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+
+* Assignments with similar descriptions that differ in the amount of whitespace between words are considered duplicate.<br>
+  e.g. `Assignment 2` is the same as `Assignment   2`.
+
```

Comment:
I think you should mention also how Assignment 2 is not the same as Assignment2

Reply:
good point